docs/features/argument_processing.md

```diff
@@ -63,17 +68,24 @@ def do_speak(self, opts):
 
 !!! note
 
-    `cmd2` sets the `prog` variable in the argument parser based on the name of the method it is decorating. This will override anything you specify in `prog` variable when creating the argument parser.
+    `cmd2` sets the `prog` variable in the argument parser based on the name of the method it is decorating.
+    This will override anything you specify in `prog` variable when creating the argument parser.
 
-As of the 3.0.0 release, `cmd2` sets `prog` when the instance-specific parser is created, which is
-later than it did previously.
+    As of the 3.0.0 release, `cmd2` sets `prog` when the instance-specific parser is created, which is later
+    than in previous versions.
 
 ## Help Messages
 
 By default, `cmd2` uses the docstring of the command method when a user asks for help on the
 command. When you use the `@with_argparser` decorator, the docstring for the `do_*` method is used
 to set the description for the `argparse.ArgumentParser`.
 
+!!! tip "description and epilog fields are rich objects"
+
+    While the `help` text itself is simply a string, both the `description` and `epilog` can contain
```

Comment:
Consider this rewording.

While the `help` field is simply a string, both the `description` and `epilog` fields can accept any [rich](https://github.com/Textualize/rich) renderable. This allows you to include all of rich's built-in objects like `Text`, `Table`, and `Markdown`.